README.md

```diff
@@ -22,7 +22,7 @@ https://github.com/getoutreach/concourse-example/blob/master/k8s/manifests/deplo
 ```Bash
 kubecfg \
 --jurl http://k8s-clusters.outreach.cloud/ \
---jurl https://raw.githubusercontent.com/getoutreach/jsonnet-libs/master \
+--jurl https://jsonnet-libs.outreach.cloud/ \
```

Comment:
Please add example to README how if somebody wants to use jsonnet-libs from particular branch they can still use github URL